src-tauri/src/repair_and_verify/mod.rs

```diff
@@ -40,26 +40,29 @@ pub fn clean_up_download_folder(
     game_install: &GameInstall,
     force: bool,
 ) -> Result<(), anyhow::Error> {
-    // Get download directory
-    let download_directory = format!(
-        "{}/___flightcore-temp-download-dir/",
-        game_install.game_path
-    );
+    const TEMPORARY_DIRECTORIES: [&str; 2] = [
+        "___flightcore-temp-download-dir",
+        "___flightcore-temp/download-dir",
```

Comment:
from the top of my head, this will result in `___flightcore-temp/` being leftover afterwards